### PR TITLE
Main menu

### PR DESCRIPTION
**Pull request by Sspadea**
## Summary ##

**Trello card**: https://trello.com/c/kVPwpJzU/11-n-41-table-ui-mock-data

- Added a library view 

## Screenshots ##

Captura de Pantalla 2022-03-08 a la(s) 10 27 08

## Screencast ##

https://user-images.githubusercontent.com/85120909/157247941-69669974-7dfa-4524-8549-2da80e997089.mov

### CODE REVIEW

File: WBooks/LibraryViewCell.swift [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        let contentInset = UIEdgeInsets(top: 5, left: 20, bottom: 5, right: 20)
+        cellView.layer.frame = cellView.frame.inset(by: contentInset)
+    }
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        // Initialization code
```
Comment (NickMano) [inline WBooks/LibraryViewCell.swift]: Remove comment

File: WBooks/LibraryViewCell.swift [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    }
+
+    override func setSelected(_ selected: Bool, animated: Bool) {
+        super.setSelected(selected, animated: animated)
+
+        // Configure the view for the selected state
```
Comment (NickMano) [inline WBooks/LibraryViewCell.swift]: Same

File: WBooks/LibraryViewCell.swift (hunk 1 above)
Comment (NickMano) [reply WBooks/LibraryViewCell.swift]: If we only have a super without modify other thing, we can remove the method

File: WBooks/LibraryViewController.swift [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        tableView.register(UINib(nibName: cellIdentifier, bundle: nil), forCellReuseIdentifier: cellIdentifier)
+        tableView.backgroundColor = UIColor(hex: "#EAF6FA")
+    }
+
+    // MARK: - Table view data source
+
```
Comment (NickMano) [inline WBooks/LibraryViewController.swift]: Remove space between the mark and the method

File: WBooks/LibraryViewController.swift [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 10
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        // #warning Incomplete implementation, return the number of rows
```
Comment (NickMano) [inline WBooks/LibraryViewController.swift]: remove comment

**Review by NickMano — APPROVED** [review]

**Review by dfmm472 — APPROVED** [review]
As a suggestion, I think that the Pods folder should not be uploaded to the repository, since with the podfile they are installed, the rest seems very good to me 👍

**Pull request merged**